<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setDialog</class>
 <widget class="QDialog" name="setDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>223</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>272</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>272</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>185</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>24</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>启动模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>84</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>后台模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>54</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>视频尺寸：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>114</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开机启动：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>137</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout1">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>音频</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="videoButton">
      <property name="text">
       <string>视频</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout2">
    <item>
     <widget class="QRadioButton" name="normalButton">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>普通</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="bigButton">
      <property name="text">
       <string>大</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>143</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout3">
    <item>
     <widget class="QRadioButton" name="taskButton">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>任务栏</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="trayButton">
      <property name="text">
       <string>托盘</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout4">
    <item>
     <widget class="QRadioButton" name="yesButton">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>是</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="noButton">
      <property name="text">
       <string>否</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>144</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>缓冲时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>145</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>144</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>MS</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioButton</tabstop>
  <tabstop>videoButton</tabstop>
  <tabstop>normalButton</tabstop>
  <tabstop>bigButton</tabstop>
  <tabstop>taskButton</tabstop>
  <tabstop>trayButton</tabstop>
  <tabstop>yesButton</tabstop>
  <tabstop>noButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
